.gitignore

```diff
@@ -15,3 +15,4 @@ build
 .nyc_output
 
 .env
+.husky
```

Comment:
This is what we have in monorail. Alternatively we could commit the `.husky/pre-commit` file so all the lint-staged will be the same. The reason we didn't have this in monorail is because there's non-JS code in the monorepo and dont want a hard dep on npm / JS for making commits (which init husky does)